### PR TITLE
prueba

### PR DESCRIPTION
**Pull request by fmcabasb**

### CODE REVIEW

**Review by alberto2254t — COMMENTED** [review]
icon_vwa
#https://wefunder.com/bannerlifeinsurance/disclosures/general_info

**Review by MrMaxich96 — COMMENTED** [review]
> #192

File: readme.md
Comment (Starrcisco) [inline readme.md:1]: readme.md

File: demos/demo-chart.html [hunk 1]
```diff
@@ -46,9 +47,9 @@ <h4 id="ideas">Ideas</h4>
...
       <h4>Demos</h4>
       <ul>
-        <li><a href="/demos/demo-table.html">Table Demo</a></li>
-        <li><a href="/demos/demo-map.html">Map Demo</a></li>
-        <li><a href="/demos/demo-chart.html">Chart Demo</a></li>
+        <li><a href="./demo-table.html">Table Demo</a></li>
+        <li><a href="./demo-map.html">Map Demo</a></li>
+        <li><a href="./demo-chart.html">Chart Demo</a></li>
       </ul>
       <h4 id="use">Use</h4>
```
Comment (Rucellmai61) [inline demos/demo-chart.html:54]: .npmignore

**Comment by Kkaungmyat** [comment]
gh pr checkout 60

**Comment by Kkaungmyat** [comment]
60

File: demos/demo-chart.html (hunk 1 above)
Comment (RAFranciscoZC) [reply demos/demo-chart.html:54]: Great

**Pull request closed without merging**